src/schema/compatibility.ts

```diff
@@ -216,15 +215,13 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
       },
     ),
     bookmarks: async (source, args, context: AuthContext, info) => {
-      const firstFolderId = await getFirstFolderId(context);
       const resolver = compatFeedResolver((ctx, args, opts, builder, alias) =>
         bookmarksFeedBuilder({
           ctx,
           unreadOnly: false,
           reminderOnly: false,
           builder: builder.orderBy('bookmark.createdAt', 'DESC'),
           alias,
-          firstFolderId,
```

Comment:
Let's also remove this prop from the interface itself.

Reply:
I think I've removed it, didn't I? 👀